### PR TITLE
argocd 2.8.1

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by p-linnane** [comment]
Not the latest release yet: https://github.com/argoproj/argo-cd/releases

**Review by SMillerDev — APPROVED** [review]

**Review by SMillerDev — DISMISSED** [review]

File: Formula/a/argocd.rb [hunk 1]
```diff
@@ -2,8 +2,8 @@ class Argocd < Formula
   desc "GitOps Continuous Delivery for Kubernetes"
   homepage "https://argoproj.github.io/cd"
   url "https://github.com/argoproj/argo-cd.git",
-      tag:      "v2.8.0",
-      revision: "804d4b8ca6bc4c2cf02c5c971aa923ec5b8623f0"
+      tag:      "v2.8.1",
+      revision: "356e33ac293cf78afb9b555a65143813c61e5e80"
   license "Apache-2.0"
```
[the remark this answers is not in the document]
Comment (p-linnane) [reply Formula/a/argocd.rb:8]: Done.

**Comment by samford** [comment]
[2.8.1 was released a few minutes ago](https://github.com/argoproj/argo-cd/releases/tag/v2.8.1) and is now the "latest" release on GitHub.

**Comment by p-linnane** [comment]
The hash changed, so I've updated that.

**Review by bayandin — APPROVED** [review]

**Pull request merged**